### DISCUSSION TITLE
How to specify multiple resource pack urls as an ENV variable?

### DISCUSSION THREAD

**Discussion by zekageri** [itzg/docker-minecraft-server#3201] [Q&A]
Hi!

I'm running this server with docker and using portainer to edit things.
If I specify multiple URLS for the `RESOURCE_PACK` variable it doesn't work.
I have tried space, `|` character and `,` to split the urls but neither of this works.

![image](https://github.com/user-attachments/assets/98f09cdf-1242-4507-8f60-e2ad0627f272)

Can someone tell me how to specify multiple resource packs?
Two `RESOURCE_PACK` named ENV variable is not possible also

**Comment by itzg** [comment]
This is just a mapping of https://minecraft.wiki/w/Server.properties#resource-pack, as such there can only be one.

**Reply by zekageri** [reply]
Thank you for the answer. Is there a method to add more resource packs?

**Reply by itzg** [reply]
> Thank you for the answer. Is there a method to add more resource packs?

Not easily. You would have to download all the resource packs, unzip them all into a combined directory, re-zip it into one, and then host that file on your own web server that all of the clients can access.

**Reply by zekageri** [reply]
Oh I see. I can easily do that. I can just spin up an other container with a node.js server, put it into the same lan as my minecraft server container and make the zip file accessible by local url. Thank you very much for the idea!
